src/server.c

```diff
@@ -2702,6 +2702,14 @@ void commandAddSubcommand(struct redisCommand *parent, struct redisCommand *subc
 /* Set implicit ACl categories (see comment above the definition of
  * struct redisCommand). */
 void setImplicitACLCategories(struct redisCommand *c) {
+    /* If it's not @fast is @slow in this binary world. */
+    if (!(c->acl_categories & ACL_CATEGORY_FAST))
+        c->acl_categories |= ACL_CATEGORY_SLOW;
```

Comment:
because of this move, all commands are now marked as slow (even the fast ones).
if we move it here, we need to check `c->flags`, not `c->acl_categories`